src/Sys/InspectSeoHandler.php

```diff
@@ -85,27 +87,30 @@ public function handleCommand(Message $command, Pbjx $pbjx): void
         }
     }
 
+    public function resolveGoogleSiteUrl(Message $node): string {
+        $url = UriTemplateService::expand(
+            "{$node::schema()->getQName()}.canonical", $node->getUriTemplateVars()
+        );
+
+        $host = parse_url($url, PHP_URL_HOST);
+        preg_match('/(?P<domain>[a-z0-9][a-z0-9\-]{1,63}\.[a-z\.]{2,6})$/i', $host, $matches);
```

Comment:
@digitalartcrew i'd recommend avoiding regex wherever possible, i think in this case if you split the hostname by `.` then the last item in that array is the tld and the penultimate is the apex